schedule/kernel/nvidia-sle.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: nvidia-sle
+schedule:
+- autoyast/prepare_profile
+- installation/bootloader_start
+- autoyast/installation
+- autoyast/console
+- autoyast/login
+- autoyast/wicked
+- autoyast/repos
+- autoyast/clone
+- autoyast/logs
+- console/yast2_vnc
+- console/force_scheduled_tasks
+- shutdown/grub_set_bootargs
+- autoyast/autoyast_reboot
+- installation/handle_reboot
+- installation/first_boot
+- qa_automation/patch_and_reboot
+- kernel/nvidia
+- console/system_prepare
```

Comment:
I thnk that the `nvidia` test should be after at least `system_prepare` because it uses serial terminal. Mabe we could also drop some of the other console setup modules?

Reply:
If we move after, it will break later x11 tests, as some changes in system_prepare will not survive reboot, i already tried that. I was thinking to remove some of them while i was copying schedule of qam-gfxdrivers which we have for  i915 .

Also, if we look on Nvidia test itself, it boots to gnome after each restart.  What about to drop everything after nvidia test? If driver was broken, it wouldn't start X, vnc would be broken already.

Reply:
If you mean this failure, the job was missing `system_prepare` entirely.
https://openqa.suse.de/tests/17874369#step/keymap_or_locale_x11/10

I don't see anything in the module that would get undone by reboot.

Reply:
That one is not related,  it is  this one https://openqa.suse.de/tests/17875481.

Reply:
I guess that the nvidia test should call `ensure_serialdev_permissions` after the last reboot.

Reply:
OK, never mind then. Those permissions should probably be set by the console backend anyway...